### ISSUE TITLE
Parking access quest and residential parking

### ISSUE THREAD

**Issue by peternewman** [streetcomplete/StreetComplete#2662]
IMHO the current parking access quest is rather ambiguous for residential parking:

- Is it open to the public (with or without fee) - Yes, you just have to buy a parking permit for the period you want to use it
- It is private - Yes, only people with a parking permit can use it (often this is only for certain times of day, so it's probably public as anyone can use it at other times
- It is only for customers - I've paid the council for a permit so I'm their customer?

The wiki doesn't seem much clearer aside from:
https://wiki.openstreetmap.org/wiki/Parking#Private.2Fresidential_parking
https://wiki.openstreetmap.org/wiki/Key:parking:lane#Residential_permits

Although I note taginfo has these which all feel relevant:
permit - 27,844
residents - 2,306
residential - 247
https://taginfo.openstreetmap.org/keys/?key=access#values

If someone can confirm the correct tag perhaps we can improve the text.

Looks like I'm not the only one (which was the cycling quest, but same choice of answers):
For "It is open to the public, but there's a yearly fee and a _huge_ waiting list to get one of the spots" cases, should I just mark it "private?" "Public" only means walk-ins welcome, right?

_Originally posted by @michaelblyons in https://github.com/streetcomplete/StreetComplete/issues/2517#issuecomment-767039451_

**How to Reproduce**
Answer the parking access quest for some residential parking.

**Versions affected**
v31.0

**Comment by michaelblyons** [comment]
You're talking about a sign that says something like this, right?

> **Street parking for residents only**
> 7 PM - 6 AM
> Permit zone 7D

I've been assuming that parking lots & structures for apartment buildings are `private`.

**Comment by peternewman** [comment]
> You're talking about a sign that says something like this, right?

Yep, well the UK ones often just say "Permit holders only". A lot are also differently restricted than yours (re-reading your hours), say 10-11 and 2-3 just to stop commuters parking there (which yours wouldn't do), but anyone could use it while shopping outside those times.

> I've been assuming that parking lots & structures for apartment buildings are `private`.

Yeah that seems a no-brainer, I'm not going to be able to park at an appartment building while I shop; it's only useful if I'm visiting a friend and it has visitors spots. These seem far more useful though.

**Comment by westnordost** [comment]
Well, "permit" is the only value that is documented. You know how it works, we cannot simply invent new tags in this issue tracker. They must at least be documented and have some usage. `access=permit` checks the boxes here.
What text would you propose for this option?

**Comment by peternewman** [comment]
> What text would you propose for this option?

How about:
"It is only for permit holders"

I guess we don't need the residents bit, as it may be other types of permit too.

Although this does potentially conflict with this wiki example:
https://wiki.openstreetmap.org/wiki/Parking#Private.2Fresidential_parking

Also this specific combination isn't listed here:
https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dparking#Additional_Tags

Double-checking the sign that triggered this, it is just for permit holders, presumably as it's a main road so they can't do roadside parking.

However most of the stuff near me is a greyer area like @michaelblyons example where it's only restricted some of the time, which I think would still be https://wiki.openstreetmap.org/wiki/Conditional_restrictions#Does_not_apply_to: or https://wiki.openstreetmap.org/wiki/Key:parking:lane

**Comment by FloEdelmann** [comment]
Related: #3429 and #3473

**Comment by matkoniecz** [comment]
Note that access=permit is for cases where you need permit and permit is generally obtainable. If only people living in nearby apartments/houses are allowed to buy a permit, then it is `access=private`

**Comment by michaelblyons** [comment]
Three or four years ago, I went to the government office of a nearby town to see if I could obtain a permit to park in a space near the transit I use (for bad weather---usually I bicycle)...

> "Sure. It is 120/year, but there is a waiting list. Do you want me to put you on it?"
>
> "Yes, please." Then a thought struck me: "How long is this list?"
>
> "Fifteen people," she says.
>
> "And how fast does it move?"
>
> "Well, last year one person got a sp---"
>
> _"FIFTEEN YEARS!? It would take fifteen YEARS to get a space?"_

Parking permits are "available" to the public, but practically unobtainable. Amusingly, this is actually a different case than the [bicycle parking comment](https://github.com/streetcomplete/StreetComplete/pull/2517#issuecomment-767039451) I made that sparked this thread (about a different place).
